ompi/mca/pml/ob1/pml_ob1_sendreq.h

```diff
@@ -385,7 +386,9 @@ mca_pml_ob1_send_request_start_btl( mca_pml_ob1_send_request_t* sendreq,
     }
 #endif /* OPAL_CUDA_GDR_SUPPORT */
 
+    //SPC_BIN_RECORD(OMPI_SPC_P2P_MESSAGE_SIZE, size);
```

Comment:
remove

Reply:
I removed these temporarily to see if they had a major impact on overhead.  I have added them back in.